pom.xml

```diff
@@ -982,21 +983,6 @@
       <id>quality</id>
       <build>
         <plugins>
-          <plugin>
-            <artifactId>maven-surefire-plugin</artifactId>
-            <configuration>
-              <argLine>
-                -Xmx300m
-                -XX:+HeapDumpOnOutOfMemoryError
-                -XX:HeapDumpPath=${project.build.directory}/surefire-reports
-                ${argLine}
-              </argLine>
-              <excludes>
-                <exclude>**/Abstract*</exclude>
-                <exclude>**/*$*</exclude>
-              </excludes>
-            </configuration>
-          </plugin>
           <plugin>
             <artifactId>maven-failsafe-plugin</artifactId>
```

Comment:
I know it's a bit unrelated, but do you think it's a good idea to remove the failsafe configuration as well?

Reply:
I would have to go through the pom files to ensure it is OK.

Therefore I would do this separately.